tests/cnf/core/network/metallb/tests/bgp-unnumbered.go

```
@@ -254,7 +255,7 @@ var _ = Describe("BGP Unnumbered", Ordered, Label(tsparams.LabelBGPUnnumbered),
 					ipAddressPoolRange)
 			})
 
-		It("Verify IBGP peering established with configured peerASN and unnumbered and BFD",
+			It("Verify IBGP peering established with configured peerASN and unnumbered and BFD",
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)